src/screens/Home/ResourcesItemScreen.js

```diff
@@ -42,6 +51,19 @@ function ResourceStep({
   );
 }
 
+/* Returns a single map marker */
+function MapMarker({
+  title, description, latitude, longitude,
+}) {
+  return (
+    <MapView.Marker
+      coordinate={{ latitude, longitude }}
+      title={title}
+      description={description}
+    />
+  );
+}
+
```

Comment:
I'm not sure this component is necessary since you don't really add anything new to `MapView.Marker`